app/javascript/pages/howToApply/how-to-apply.tsx

```diff
@@ -125,7 +129,38 @@ const eligibilityListItems = [
   { index: 5 },
 ]
 
-const BeforeYouStartSection = () => {
+const LeasingAgentBox = ({ listing }: { listing: RailsSaleListing }) => {
```

Comment:
suggestion: If we move this component to its own file or export it, it might make testing it possible and remove the codeclimate error. I see value in testing this section since there is conditional logic on how the component should appear depending on the backend data available.